docs/devsettings.md

```diff
@@ -0,0 +1,25 @@
+---
+id: devsettings
+title: DevSettings
+---
+
+`DevSettings`
+
+---
+
```

Comment:
Could we add a summary here? Something like "The DevSettings module exposes methods for customizing settings in for developers in development."